src/controllers/QueryRunner.ts

```diff
@@ -198,6 +173,7 @@ export default class QueryRunner {
 
         // Store the batch again to get the rest of the data
         this._batchSets[batch.id] = batch;
+        this._totalElapsedMilliseconds += new Date(batch.executionEnd).getTime() - new Date(batch.executionStart).getTime();
```

Comment:
Doesn't a batch come with ExecutionElapsedTime property already? Why would be calculate it in here?